src/bidiMapper/domains/network/NetworkProcessor.ts

```diff
@@ -66,8 +66,38 @@ export class NetworkProcessor {
     };
   }
 
-  continueRequest(_params: Network.ContinueRequestParameters): EmptyResult {
-    throw new UnknownCommandException('Not implemented yet.');
+  async continueRequest(
+    params: Network.ContinueRequestParameters
+  ): Promise<EmptyResult> {
+    const networkId = params.request;
+    const blockedRequest = this.#getBlockedRequest(networkId);
+    const {request: fetchId, phase} = blockedRequest;
+
+    if (phase !== Network.InterceptPhase.BeforeRequestSent) {
+      throw new InvalidArgumentException(
+        `Blocked request for network id '${networkId}' is not in 'BeforeRequestSent' phase`
+      );
+    }
+
+    if (params.url !== undefined) {
+      NetworkProcessor.parseUrlString(params.url);
+    }
+
+    const {url, method} = params;
+    // TODO: Set / expand.
+    // ; headers
+    // ; cookies
+    // ; body
+
+    await this.#networkStorage
+      .getRequest(networkId)
+      ?.continueRequest(fetchId, url, method);
+
+    this.#networkStorage.removeBlockedRequest(networkId);
+
+    // TODO: Remove from network request map?
```

Comment:
I don't think so, as the request still exists.

Reply:
Done (will push later)